cmd/lakectl/cmd/merge.go

```diff
@@ -37,12 +37,13 @@ var mergeCmd = &cobra.Command{
 
 		result, err := client.Merge(context.Background(), leftRefURI.Repository, leftRefURI.Ref, rightRefURI.Ref)
 		if errors.Is(err, catalog.ErrConflictFound) {
-			DieFmt("%d conflict(s) found\n", result.Summary.Conflict)
+			_, _ = fmt.Printf("Conflicts: %d\n", result.Summary.Conflict)
+			return
 		}
 		if err != nil {
 			DieErr(err)
 		}
-		fmt.Printf("Merged %s\n", result.Reference)
+		_, _ = fmt.Printf("new: %d modified: %d removed: %d\n", result.Summary.Added, result.Summary.Changed, result.Summary.Removed)
```

Comment:
Can we also have the commit id (Reference)?

Reply:
Good idea.  Added it, and moved to a proper template and all, and even added some more fields to be (a bit) more like `lakectl commit`.